src/Helpers.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+function registerSpatiePestHelpers(): void
+{
+    function whenConfig(string $key): void
+    {
+        if (config($key) === null) {
+            test()->markTestSkipped("{$key} is not set in the config file");
+        }
+    }
+
+    function skipOnGitHubActions(): void
+    {
+        if (getenv('GITHUB_ACTIONS') === 'true') {
+            test()->markTestSkipped('This test is skipped on GitHub Actions');
+        }
+    }
+
+    function whenGitHubActions(): void
+    {
+        if (getenv('GITHUB_ACTIONS') === 'true') {
```

Comment:
```suggestion
        if (getenv('GITHUB_ACTIONS') !== 'true') {
```

I think this is meant to be negated, when we're not in GitHub Actions we want to skip the test.